modules/apps/saved-content/saved-content-web/src/main/resources/META-INF/resources/my_saved_content/view.jsp

```diff
@@ -16,4 +16,73 @@ if (Validator.isNotNull(backURL)) {
 	portletDisplay.setShowBackIcon(true);
 	portletDisplay.setURLBack(backURL);
 }
-%>
\ No newline at end of file
+
+MySavedContentDisplayContext mySavedContentDisplayContext = new MySavedContentDisplayContext(liferayPortletRequest, liferayPortletResponse);
+%>
+
+<clay:container-fluid>
+	<liferay-ui:search-container
+		emptyResultsMessage="no-saved-content-were-found"
+		searchContainer="<%= mySavedContentDisplayContext.getSearchContainer() %>"
+	>
+		<liferay-ui:search-container-row
+			className="com.liferay.saved.content.model.SavedContentEntry"
+			escapedModel="<%= true %>"
+			keyProperty="savedContentEntryId"
+			modelVar="savedContentEntry"
+		>
+
+			<%
+			String assetTitle = mySavedContentDisplayContext.getAssetTitle(savedContentEntry.getClassName(), savedContentEntry.getClassPK());
+			%>
+
+			<liferay-ui:search-container-column-text
+				cssClass="table-cell-expand"
+			>
+				<p class="list-group-title"><%= assetTitle %></p>
+				<p class="list-group-subtitle"><%= ResourceActionsUtil.getModelResource(locale, savedContentEntry.getClassName()) %></p>
+			</liferay-ui:search-container-column-text>
+
+			<liferay-ui:search-container-column-text
+				cssClass="table-cell-expand-smallest"
+			>
+				<div class="btn-group">
```

Comment:
This looks a bit strange, putting two links inside a `btn-group-item`, that looks more for buttons. 
If you just want to separate the links, remove those `divs` and set a class `mr-2`(or 1 or 3, I don't know how much should be) to the first link, or `ml-X` to the second link. Reach me by slack if this is not clear.

Reply:
@ambrinchaudhary 
Can you recheck the last commit if this is what you were meaning?

cc/ @boton 

<img src="https://media2.giphy.com/media/3oz8xIsloV7zOmt81G/giphy.gif"/>

Reply:
I think that this wrapping with `<div class="btn-group">` is good enough for links if we are using them as buttons like [in this case](https://github.com/liferay-lima/liferay-portal/pull/4911/files/ee9ab832afcf56e4f1ca8257f4f42521916d3e5a#diff-eb889b838b0426ec3d0dce86ce5efe2a899622f9c39aeb84a135be553c53b59bR62).

There are some uses but not many across the portal
![image](https://github.com/liferay-lima/liferay-portal/assets/67901/f15bdef3-a265-4abd-9f2e-6a5f45d4018d)


But anyway I'm happy with the `mr-x` solution too.